src/assets/themes/red-theme.css

```
@@ -0,0 +1,167 @@
+ * ===============================*/
+
+/* Breathing dot animation - red glow */
+.breathing-dot-inner {
+  background-color: var(--red-500) !important;
+  box-shadow: 0 0 40px var(--red-500) !important;
+}
+
+@keyframes breathe {
+  0%,
+  100% {
+    box-shadow: 0 0 10px var(--red-500) !important;
```

Biome (2.1.2)

[error] Using !important within keyframes declaration is completely ignored in some browsers.

Consider removing useless !important declaration.

(lint/suspicious/noImportantInKeyframe)